app/Jobs/Server/UpdateKernelJob.php

```
@@ -0,0 +1,59 @@
+        );
+    }
+
+    private function broadcastServerUpdate(): void
+    {
+        $this->server->refresh();
+
+        SocketEvent::dispatch(new SocketEventDTO(
+            projectId: $this->server->project_id,
+            type: 'server.updated',
+            data: new ServerResource($this->server),
+        ));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Events\SocketEvent' in method 'broadcastServerUpdate'. (undefined)

(StaticAccess)